packages/core/test/unit/query-generator/select-query.test.ts

```diff
@@ -491,8 +503,6 @@ Only named replacements (:name) are allowed in literal() because we cannot guara
         attributes: [
           // these used to have special escaping logic, now they're always escaped like any other strings. col, fn, and literal can be used for advanced logic.
           ['count(*)', 'count'],
-          '.*',
-          '*',
```

Comment:
Those are important to keep as they test against a vulnerability that used to exist. I'd prefer to use `// @ts-expect-error` here

Reply:
I've readded this now